### PR TITLE
Update: String format from .format() to f

### PR DESCRIPTION
**Pull request by Eugene-Oluoch**

### CODE REVIEW

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
Did you make these changes with [`pyupgrade`](https://pypi.org/project/pyupgrade) or [`flynt`](https://pypi.org/project/flynt) or manually?

**Comment by Eugene-Oluoch** [comment]
I did them manually

**Review by cclauss — APPROVED** [review]
LGTM

You can _automate_ these fixes with [`ruff --select=UP032 --fix .`](https://beta.ruff.rs/docs/rules/#pyupgrade-up).

**Comment by Eugene-Oluoch** [comment]
> 

Sure thank you

**Comment by cclauss** [comment]
Git conflicts to resolve.

File: core.py [hunk 1]
```diff
@@ -68,8 +68,8 @@ def show_options(self, parent = None):
         for index, option in enumerate(self.OPTIONS):
             print(f"[{index + 1}] {option[0]}")
         if self.PROJECT_URL:
-            print(f"[{98}] Open project page")
-        print(f"[{99}] Back to {parent.TITLE if parent is not None else 'Exit'}")
+            print('[98] Open project page')
+        print(f"[99] Back to {parent.TITLE if parent is not None else 'Exit'}")
```
Comment (cclauss) [inline core.py:72]:
```suggestion
        print(f"[99] Back to {parent.TITLE if parent else 'Exit'}")
```

File: core.py [hunk 2]
```diff
@@ -157,7 +157,7 @@ def show_options(self, parent = None):
         self.show_info()
         for index, tool in enumerate(self.TOOLS):
             print(f"[{index} {tool.TITLE}")
-        print(f"[{99}] Back to {parent.TITLE if parent is not None else 'Exit'}")
+        print(f"[99] Back to {parent.TITLE if parent is not None else 'Exit'}")
```
Comment (cclauss) [inline core.py:160]:
```suggestion
        print(f"[99] Back to {parent.TITLE if parent else 'Exit'}")
```

**Review by cclauss — APPROVED** [review]
Nice!  Thanks.

**Pull request closed without merging**